pkg/namesgenerator/names-generator.go

```diff
@@ -559,6 +562,9 @@ var (
 
 		// Christiane Nüsslein-Volhard - German biologist, won Nobel Prize in Physiology or Medicine in 1995 for research on the genetic control of embryonic development. https://en.wikipedia.org/wiki/Christiane_N%C3%BCsslein-Volhard
 		"volhard",
+		
+		// Cédric Villani - French mathematician, won Fields Medal, Fermat Prize and Poincaré Price for his work in differential geometry and statistical mechanics
```

Comment:
Can you add a link with more information as well (e.g. to wikipedia, as is used for other names?)